certbot-auto

```diff
@@ -131,7 +131,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO+x}" ]; then
```

Comment:
Thanks for the explanation. I only tested with a defined variable. Is that meant to support being called by some other script that has pre-set some variables? I'll have to dig around and look at the rest of this codebase.